src/commons/UI/BaseModal.vue

```diff
@@ -32,9 +31,10 @@ export default defineComponent({
       type: String
     }
   },
+  emits: ["close"],

```

Comment:
Note: Use "close" instead because when use it will be
page --> modal --> base modal

When base modal emits CLOSE_MODAL, it will sometimes emit to page and modal, sometimes only emit to modal. Therefore, I let base-modal emits "close" to modal & when modal receives it, it will emits CLOSE_MODAL to page where page will remove the modal component.